content/articles/data-representation-using-brython/index.md

```diff
@@ -0,0 +1,337 @@
+The cooperation between JavaScript and the web has been a standard. There are different programming languages used for web development, with a notable mention being Python.  Python has micro-frameworks such as `Flask`, `Hug`, and `Cherrypy` which have enabled Python to cooperate with the web.
+
+Instead of using Python micro-frameworks to run the client-side web applications, we have `Brython`. `Brython` stands for Browser Python.  `Brython` is a client-side scripting language with Python3 execution that is adjusted to the HTML5 environment. 
+
+Brython interfaces smoothly with JavaScript and its libraries to build high-level web projects. It incorporates libraries such as Vue.js, Bokeh, Highchart, and Phaser.
+
+In this tutorial, we will represent data utilizing piecharts and scatterplots in Brython. This is possible because of the cooperation among Brython and Highchart.js libraries. Let's get started!
+### Prerequisites
+
+To follow along easily, the reader should:
+
+- Install the [Python](https://www.python.org/downloads/) executable.
+-  Have [Python programming](/engineering-education/python-projects-for-beginners/) skills.
+-  Have [Javascript programming](https://www.w3schools.com/js/) skills.
+-  Have [Visual Studio Code.](https://code.visualstudio.com/)
+
+### Aim of this tutorial
+In this tutorial, we aim to guide the readers on how to:
+
+- Installing Brython in our local environment. We can achieve this using a host of methods:
+    1. PyPI. To use this method make sure you have installed Python3.
+    2. Content Delivery Network (CDN).
+    3. Node Package Manager (npm).
+    4. GitHub.
+     
+> NOTE: We will focus on the second and third methods in creating a piechart and scatter plot.
+
+### Getting started
+
+#### Installing brython with the node package manager
+Node.js is a JavaScript runtime used for building scalable network applications and executes JavaScript code outside a web browser. 
+To install Brython using npm, [Node.js](https://nodejs.org/en/) must be downloaded and installed. After installation, type the following command on your terminal:
+```bash
+npm install brython
+```
+Do this in your preferred directory for your project. You will notice a folder called node_modules, and within it contains the Brython package. 
+Thus, you can add it to your web project using the script tag below:
```

Comment:
```suggestion
Run this command while in your working directory. This will create a folder called node_modules. This folder contains the Brython package.

To add the Brython package to your web project, use the script tag below:
```